test/MessageTraitTest.php

```diff
@@ -305,31 +305,6 @@ public function testNumericHeaderValues()
         ];
     }
 
-    /**
-     * @dataProvider testNumericHeaderValues
-     * @group 99
-     */
-    public function testFilterHeadersShouldAllowIntegersAndFloats($value)
```

Comment:
This goes to another comment I've made in the review: numeric values should be allowed, which is what #99 was reporting. Internally, we can convert those to strings, so that the return values of `getHeaders()`, `getHeader()`, and `getHeaderLine()` are consistent with the spec. But we *must* allow them; requiring developers to cast those values to strings manually before calling `withHeader()` is a BC break, and poor for usability.

This test will likely need to be _rewritten_, but the behavior we're testing is the same: numeric values should be valid for header values.